js/tab_panel.ts

```diff
@@ -0,0 +1,183 @@
+import { html, css, nothing, LitElement, PropertyValues } from "lit";
+import { property, queryAll, queryAssignedElements } from "lit/decorators.js";
+import { legacyStyles } from "./ipywidgets_styles";
```

Comment:
The imports aren't in order. Could you move the local imports below the lit imports and add a space between?

Reply:
Thank you! Done.